.github/workflows/ci-lint.yml

```diff
@@ -0,0 +1,44 @@
+name: Code linting
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+defaults:
+  run:
+    shell: bash
```

Comment:
This can be removed since it's only using Linux. It was needed for macOS/Windows